### ISSUE TITLE
Prior helm install of stable/prometheus-operator chart not reflected in helm-diff output

### ISSUE THREAD

**Issue by mojochao** [databus23/helm-diff#168]
I have installed the `stable/prometheus` chart into my cluster with `helm install prometheus-operator stable/prometheus-operator` command with helm v3 and it shows up in my release listing.

```shell
$ helm version
version.BuildInfo{Version:"v3.0.0", GitCommit:"e29ce2a54e96cd02ccfce88bee4f58bb6e2a28b6", GitTreeState:"clean", GoVersion:"go1.13.4"}

$ helm plugin list
NAME    	VERSION   	DESCRIPTION
2to3    	0.2.0     	migrate and cleanup Helm v2 configuration and releases in-place to Helm v3
diff    	3.0.0-rc.7	Preview helm upgrade changes as a diff
helm-git	0.5.0     	Get non-packaged Charts directly from Git.
registry	0.7.0     	This plugin provides app-registry client to Helm.
        	          	usage:
        	          	  $ helm registr...

$ helm list -n monitoring
NAME               	NAMESPACE 	REVISION	UPDATED                             	STATUS  	CHART                    	APP VERSION
prometheus-operator	monitoring	1       	2019-11-25 14:59:45.309477 -0600 CST	deployed	prometheus-operator-8.2.4	0.34.0
```

Running a `helm -n monitoring diff upgrade prometheus-operator stable/prometheus-operator --reuse-values` command shows a diff containing adds of resources already present in the cluster from the prior `helm install prometheus-operator stable/prometheus-operator` command, and I expected to see no diff.

This is the only chart in my cluster that behaves this way when a `helm diff upgrade` command is run for them.

Are my expectations wrong or is this a bug?  Does the helm-diff plugin correctly handle helm v3 namespace-scoped releases?  It certainly seems to for all other charts I've used.

Many thanks in advance for any info or diagnostic advice!

**Comment by databus23** [comment]
I believe the underlying problem is the same as reported here: #165

The prometheus-operator chart makes use of `.Capabilites` values which currently are broken in the diff plugin for helm v3:
https://github.com/helm/charts/blob/master/stable/prometheus-operator/templates/grafana/dashboards-1.14/k8s-coredns.yaml#L2

The problem is not with the plugin but actually with building `helm template` command which `diff` uses under the hood.
I addressed this issue with this PR: https://github.com/helm/helm/pull/7054. Check the comments on this PR for some more details about the issue.
So this can be fixed as soon as helm 3.1.0 is released.

**Comment by databus23** [comment]
I believe this is has been resolved in the latest release of this plugin.
